targets/targets.json

```diff
@@ -6844,29 +6844,52 @@
         "extra_labels_add": [
             "PSOC6_02",
             "MXCRYPTO_02",
-            "CORDIO"
+            "CORDIO",
+            "TFM",
+            "TFM_DUALCPU"
         ],
         "macros_add": [
             "CYB0644ABZI_S2D44",
             "CY_IPC_DEFAULT_CFG_DISABLE",
-            "CYBSP_WIFI_CAPABLE"
+            "CYBSP_WIFI_CAPABLE",
+            "TFM_MULTI_CORE_MULTI_CLIENT_CALL=1"
         ],
         "detect_code": [
             "1910"
         ],
         "forced_reset_timeout": 5,
-        "hex_filename": "psoc6_02_cm0p_secure.hex",
+        "hex_filename": "tfm_s.hex",
         "cm0_img_id": 1,
         "cm4_img_id": 16,
-        "boot_scheme": "single_image",
-        "policy_file": "policy_single_CM0_CM4.json",
+        "boot_scheme": "multi_image",
+        "policy_file": "policy_multi_CM0_CM4_tfm.json",
         "post_binary_hook": {
             "function": "PSOC6Code.sign_image"
         },
         "overrides": {
             "network-default-interface-type": "WIFI"
         },
-        "program_cycle_s": 10
+        "program_cycle_s": 10,
+        "tfm_target_name": "psoc64",
+        "tfm_bootloader_supported": false,
+        "tfm_default_toolchain": "GNUARM",
+        "tfm_supported_toolchains": [
+            "GNUARM"
+        ],
+        "tfm_delivery_dir": "TARGET_Cypress/TARGET_PSOC6/TARGET_CY8CKIT_064B0S2_4343W",
+        "TFM_OUTPUT_EXT": "hex"
+    },
+    "CY8CKIT_064B0S2_4343W_NPSA": {
+        "inherits": [
+            "CY8CKIT_064B0S2_4343W"
+        ],
+        "features_remove": [
+            "PSA"
+        ],
+        "extra_labels_remove": [
+            "TFM",
+            "TFM_DUALCPU"
+        ]
```

Comment:
Do we need to add `components_add` "CM0P_SECURE" and `components_remove` CM0P_SLEEP for the non-PSA target like before? We may also need to add to the NPSA target:
```
"hex_filename": "psoc6_02_cm0p_secure.hex",
"boot_scheme": "single_image",
"policy_file": "policy_single_CM0_CM4.json",
```